compiler/rustc_metadata/src/creader.rs

```diff
@@ -402,7 +404,7 @@ impl<'a, 'tcx> CrateLoader<'a, 'tcx> {
     fn register_crate(
         &mut self,
         host_lib: Option<Library>,
-        root: Option<&CratePaths>,
+        dep_root: Option<&CratePaths>,
```

Comment:
I don't see the point of replacing `root` with `def_root` here, it might be simpler to keep the previous one

Reply:
The changes to this file are somewhat unrelated now, they were more relevant for the initial implementation. But I still think this makes sense, even if off topic - before this change, `root` could sometimes refer to a `CrateRoot` and sometimes to the dependency root's `CratePaths` (like here), it was pretty confusing to track through the call stack because the meaning switched back and forth. So this patch just makes the same name always refer to the same thing.

Reply:
That's OK